CheckManga/checkmanga.py

```diff
@@ -0,0 +1,128 @@
+import requests
+import sqlite3
+import bs4 as bs
+import os
+from flask import Flask, render_template, make_response, jsonify, g
+
+
+app = Flask(__name__)
+
+app.config.from_object(__name__)
+app.config.update(dict(
+    DATABASE=os.path.join(app.root_path, 'mycheckmanga.db'),
+    SECRET_KEY='development key',
+    DEBUG=True,
+    #USERNAME='admin',
+    #PASSWORD='default'
+))
+
+@app.route('/')
+def main_page():
+    """Returns the main page for the check_manga site."""
+    return render_template('main_page.html')
+
+@app.route('/checkmanga/', methods=['POST'])
+def check_manga():
+    """Connects to the db, looks up all manga marked as 'Ongoing' scrapes all of those titles, returns their status as JSON.
+    Input: The cursor object and an auto_update boolean (default=True) that tells the function whether to automatically update LastChapterRead to be the same as MostRecentChapter.
+    Output: None. Updates the DB by scraping for chapters for manga where Status = 'Ongoing' and where MostRecentChapter = LastChapterRead"""
+
```

Comment:
how is this function not syntax erroring? theres no body here

Reply:
I should have deleted this file/left it out of the commit my bad.